concepts/analysis_intro.md

```diff
@@ -1,6 +1,43 @@
-# Work in Progress
+Getting Started with Firefox Data
+=================================
 
-This article is under development.
-The work is being tracked in 
-[this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341706).
+Firefox clients out in the wild send us data as *pings*. [http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/main-ping.html](Main pings) contain some combination of *environment* data (e.g. operating system, hardware, Firefox version), *measurements* (e.g. max number of open tabs, time spent running in Javascript garbage collection), and *events* (a work in progress). We have quite a few different pings, but most of our data for Firefox Desktop comes in from main pings.
```

Comment:
It looks like these links are reversed. Syntax for a link is `[text](url)` but these appear to be `[url](text)`. You can verify that these links render correctly by running `gitbook serve` locally.

Please fix here and throughout.